search-provider/eks-article-metadata-query-provider.c

```diff
@@ -0,0 +1,433 @@
+/* Copyright 2016 Endless Mobile, Inc. */
+
+#include "ekn-query-util.h"
+#include "eks-provider-iface.h"
+
+#include "eks-knowledge-app-dbus.h"
+#include "eks-article-metadata-query-provider.h"
+#include "eks-article-metadata-query-provider-dbus.h"
+
+#include <eos-knowledge-content.h>
+#include <eos-shard/eos-shard-shard-file.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+struct _EksArticleMetadataProvider
+{
+  GObject parent_instance;
+
+  gchar *application_id;
+  EksArticleMetadata *skeleton;
+  GCancellable *cancellable;
```

Comment:
Nothing ever creates or sets this

Reply:
Indeed. This was a leftover from copy-pasting the other provider.